app/src/main/java/com/ruslanchikbananchik/rmma/common/Common.kt

```diff
@@ -0,0 +1,12 @@
+package com.ruslanchikbananchik.rmma.common
+
+import com.ruslanchikbananchik.rmma.`interface`.RetrofitServices
+import com.ruslanchikbananchik.rmma.retrofit.RetrofitClient
+import retrofit2.Retrofit
+import retrofit2.create
+
+object Common {
+    private val BASE_URL = "https://meowfacts.herokuapp.com/"
+    val retrofitService: RetrofitServices
+        get() = RetrofitClient.getClient(BASE_URL).create(RetrofitServices::class.java)
```

Comment:
Why did you use `get()` here? With this `get()` a new instance of `RetrofitService` will be created on each call. You don't need that.

You tried to hide implementation details of `RetrofitService` creation here. This approach is called "Factory method" pattern. You can rename this class from `Common` to `RetrofitServiceFactory` and add `fun create(): RetrofitService` function. It would be clearer for other devs who will read this code.